src/main/kotlin/com/sourcegraph/cody/history/HistoryTree.kt

```diff
@@ -160,32 +177,31 @@ class HistoryTree(
           .groupBy { chat -> DurationGroupFormatter.format(chat.getUpdatedTimeAt()) }
 
   private class LeafPopupAction(
-      private val tree: SimpleTree,
       text: String,
-      icon: Icon? = null,
+      icon: Icon?,
+      private val isEnabled: () -> Boolean,
       private val action: () -> Unit
   ) : AnAction(text, null, icon) {
+
     override fun update(e: AnActionEvent) {
-      super.update(e)
```

Comment:
Is removal of `super.update(e)` on purpose? What does it change?

Reply:
Implementation of `super.update(e)`:

```java
    public void update(@NotNull AnActionEvent e) {
        if (e == null) {
            $$$reportNull$$$0(10);
        }

    }
```

So is basically non-null assertion. We don't have to do that because `AnActionEvent` is non-null in our Kotlin code (not `AnActionEvent?`). That's why i believe i can remove that.